test/verify/check-storage-stratis

```diff
@@ -496,7 +519,11 @@ systemctl restart stratisd
         b.click(overprov_toggle)
         b.wait_visible(f"{overprov_toggle}:not(:checked)")
 
-    def testFilesystemSizes(self):
+    @testlib.skipImage("Stratis has no legacy yet", *V1_POOL_IMAGES)
```

Comment:
So `legacy` pools are V1 pools? which are the default on the V1_POOL_IMAGES so they are skipped when `legacy=True` because they would already be created with Cockpit.

The skipImage description doesn't really explain this well.

Reply:
Hmm, so, just "Stratis too old"?